lib/segment/src/index/field_index/mmap_point_to_values.rs

```diff
@@ -351,9 +347,9 @@ impl<T: MmapValue + ?Sized> MmapPointToValues<T> {
         if point_id < self.header.points_count as PointOffsetType {
             let range_offset = (self.header.ranges_start as usize)
                 + (point_id as usize) * std::mem::size_of::<MmapRange>();
-            self.mmap
-                .get(range_offset..range_offset + std::mem::size_of::<MmapRange>())
-                .and_then(MmapRange::read_from)
+            MmapRange::read_from_prefix(self.mmap.get(range_offset..)?)
```

Comment:
As I understand, `read_from_prefix` truncates additional bytes and we can provide longer byte slice

Reply:
In the old code we truncate additional bytes manually (line 355).